task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -107,6 +107,22 @@ spec:
 
         if __name__ == "__main__":
             main()
+    - name: store-hacbs-test-output-result
+      image: quay.io/redhat-appstudio/hacbs-test:latest
+      script: |
+        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --null-input \
+          '{result: "ERROR", timestamp: $date}')
+        if [ -f /tekton/home/clamscan-result.json ];
+        then
+          cat /tekton/home/clamscan-result.json
+          HACBS_TEST_OUTPUT=$(jq -rce --arg date $(date +%s) \
+            '. | { result: (if (.infected_files != "0") then "FAILURE" else "SUCCESS" end),
+                    timestamp: $date,
+                    successes: 1,
+                    failures: .infected_files
+                  }' /tekton/home/clamscan-result.json || true)
+        fi
+        echo "${HACBS_TEST_OUTPUT:-${HACBS_ERROR_OUTPUT}}" | tee $(results.HACBS_TEST_OUTPUT.path)
```

Comment:
`/tekton/scripts/script-2-7vng4: line 15: results.HACBS_TEST_OUTPUT.path: command not found`

Task has no result defined.